ale_linters/swift/swiftlint.vim

```diff
@@ -0,0 +1,9 @@
+" Author: David Mohundro <david@mohundro.com>
+" Description: swiftlint for swift files
+
+call ale#linter#Define('swiftlint', {
+\   'name': 'swiftlint',
+\   'executable': 'swiftlint',
+\   'command': g:ale#util#stdin_wrapper . ' swiftlint',
```

Comment:
The first argument to the `stdin_wrapper` command should be a file extension for the temporary file.